torch/distributed/distributed_c10d.py

```diff
@@ -341,12 +346,15 @@ def get_backend(group=group.WORLD):
     _check_default_pg()
 
     if group == GroupMember.WORLD:
+        assert _default_pg is not None
         pg = _default_pg
```

Comment:
we can replaced all of these with local variable
```suggestion
        pg = _check_default_pg()
```
if we adopt the above method, and simplify those changes below